models/BaseLucee.cfc

```diff
@@ -513,7 +513,7 @@ component accessors=true extends='cfconfig-services.models.BaseConfig' {
 				DSXMLNode.XMLAttributes[ 'class' ] = translateDatasourceClassToLucee( translateDatasourceDriverToLucee( DSStruct.dbdriver ), DSStruct.class ?: '' );
 			 }
 			if( !isNull( DSStruct.dbdriver ) ) {
-				DSXMLNode.XMLAttributes[ 'dbdriver' ] = translateDatasourceDriverToLucee( DSStruct.dbdriver );
+				DSXMLNode.XMLAttributes[ 'dbdriver' ] = translateDatasourceDriverToLucee( DSStruct.dbdriver ) == 'MSSQL2' ? 'JTDS' : translateDatasourceDriverToLucee( DSStruct.dbdriver ); // Override DB Driver name for MSSQL2
```

Comment:
If the driver name actually needs to be `JTDS` for Lucee then let's just make the translate function do that.